Resources/Prototypes/Voice/speech_emote_sounds.yml

```
@@ -36,6 +36,22 @@
     params:
       variation: 0.125
 
+- type: emoteSounds
+  id: UnisexFelinid
+  params:
+    variation: 0.125
+  sounds:
+    Scream:
+      collection: FelinidScreams
+
+- type: emoteSounds
+  id: UnisexMoth
+  sound:
+    Scream:
+      path: /Audio/Voice/Moth/scream_moth.ogg
+  params:
+    variation: 0.125
```

```suggestion
  id: UnisexMoth
  params:
    variation: 0.125
  sounds:
    Scream:
      path: /Audio/Voice/Moth/scream_moth.ogg
```
`params` before `sounds`.
`sound` needs to be `sounds` because `sound` is the single fallback and `sounds` is the collection.